concordium-contracts-common/src/schema_json.rs

```diff
@@ -964,6 +999,109 @@ impl Type {
     ) -> Result<(), JsonError> {
         write_bytes_from_json_schema_type(schema, json, out)
     }
+
+    pub fn schema_type_to_json(&self) -> serde_json::Value {
+        match self {
+            Self::Enum(enum_type) => {
+                let mut outer_map = Map::new();
+
+                let mut vector = Vec::new();
+
+                for (name, field) in enum_type.iter() {
+                    let mut inner_map = Map::new();
+                    inner_map.insert(name.to_string(), field.field_to_json());
+                    vector.push(serde_json::Value::Object(inner_map))
+                }
+                outer_map.insert("Enum".to_string(), serde_json::Value::Array(vector));
+
+                serde_json::Value::Object(outer_map)
+            }
+            Self::TaggedEnum(tagged_enum) => {
+                let mut outer_map = Map::new();
+
+                let mut vector = Vec::new();
+
+                for (_tag, (name, field)) in tagged_enum.iter() {
+                    let mut inner_map = Map::new();
+                    inner_map.insert(name.to_string(), field.field_to_json());
+                    vector.push(serde_json::Value::Object(inner_map))
+                }
+                outer_map.insert("Enum".to_string(), serde_json::Value::Array(vector));
+
+                serde_json::Value::Object(outer_map)
+            }
+            Self::Struct(field) => field.field_to_json(),
+            Self::ByteList(_) => {
+                serde_json::Value::String("<String with lowercase hex>".to_string())
+            }
+            Self::ByteArray(_) => {
+                serde_json::Value::String("<String with lowercase hex>".to_string())
+            }
+            Self::String(_) => serde_json::Value::String("<String>".to_string()),
+            Self::Unit => serde_json::Value::Array(Vec::new()),
```

Comment:
To me `null` makes more sense here

Reply:
But that is not how we encode it everywhere else.

Reply:
I kept it as it is because that is what `concordium-client` does.